### ISSUE TITLE
I seem to have screwed up badly

### ISSUE THREAD

**Issue by frogmouse** [TiddlyWiki/TiddlyDesktop#249]
I've just installed tiddlydesktop on win 10, then opened an empty tw5, named the first tiddler, then opened Stroll and drag&dropped its tiddlers.  All went well until I came to the "save, then reload" part.  I saved just fine, but, as I couldn't find a refresh or reload button, I closed tiddly desktop and reopened it.  I got the top row of buttons to choose the wiki (they didn't work) and a screenfull of js code that I couldn't affect.  I said "Oops" (well, perhaps something stronger) and after fussing with it a while, deleted everything in the program directory, everything in the wiki directory, and started over with a new zip.
Now I have a tiddlydesktop with no wiki and javascript all over it.  Here's a snip:
![TidDsktpBad](https://user-images.githubusercontent.com/30908800/161836015-32fba460-2573-48ee-9bd6-c18cb641e1b8.JPG)
**There must be something else I have to destroy before I can begin clean and fresh.**  What would it be?

**Comment by brickfrog** [comment]
Can you click the white box and press remove, then readd? I (believe) that's what worked for when it happened to me.

**Comment by frogmouse** [comment]
Yes!  That worked for me, too.  I was able to find the last backup and everything looks good.  Thank you very much.
